RTech.CoffeeMachine.Api/BrewCoffee/Models/BrewStatusResponse.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace RTech.CoffeeMachine.Api.BrewCoffee.Models;
+
+// Interviewer notes:
+// * This record is a reference type (i.e.: wont cause memory issues when used as a parameter).
```

Comment:
https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/records#characteristics-of-records